tangorb/dao/src/main/java/fr/esrf/TangoApi/DatabaseDAODefaultImpl.java

```diff
@@ -513,11 +516,11 @@ public void unexport_device(Database database, String devname)
 	 * @see fr.esrf.TangoApi.IDatabaseDAO#export_device(fr.esrf.TangoApi.DbDevExportInfo)
 	 */
     //==========================================================================
-    public void export_device(Database database, DbDevExportInfo devinfo)
+    public void export_device(Database database, DeviceExportInfo deviceExportInfo)
```

Comment:
NoSuchMethodError - This is what I was afraid of. In the long term perspective the API will change. Otherwise there is a mess of new and old classes and it is not clear what user must use (preferably classes from JTangoClientLang). So Astor has to migrate and use `org.tango.client.database.Database#exportDevice` method. I will mark fr.esrf.TangoApi.Database as deprecated but change the signature back.